### PR TITLE
Replace repetitive actions with Action Groups in AdminConfigurableProductLongSkuTest

### PR DESCRIPTION
**Pull request by kate-kyzyma**
### Description (*)
The test is refactored according to the best practices followed by MFTF.

### Manual testing scenarios (*)
Run the test in the local environment and verified that it run successfully

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by kate-kyzyma** [comment]
@magento run all tests

File: app/code/Magento/ConfigurableProduct/Test/Mftf/ActionGroup/AssertAdminChildProductDataOnParentProductEditPageActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminChildProductDataOnParentProductEditPageActionGroup">
+        <annotations>
+            <description>Proper Child product data (name, sku, price) are shown on the Configurable Product edit page.</description>
```
Comment (dmytro-ch) [inline app/code/Magento/ConfigurableProduct/Test/Mftf/ActionGroup/AssertAdminChildProductDataOnParentProductEditPageActionGroup.xml]:
```suggestion
            <description>Verify the proper child product data (name, sku, price) is shown on the Configurable Product edit page.</description>
```

**Review by dmytro-ch — CHANGES_REQUESTED** [review]

**Comment by dmytro-ch** [comment]
Hello @kate-kyzyma, could you please check and fix the issues reported by Semantic Version Checker?

**Comment by kate-kyzyma** [comment]
@magento run all tests

**Comment by kate-kyzyma** [comment]
@magento run all tests

**Comment by dmytro-ch** [comment]
@magento run Functional Tests EE, Static Tests

**Comment by dmytro-ch** [comment]
@magento run Static Tests

**Comment by dmytro-ch** [comment]
@magento run Functional Tests EE

**Comment by dmytro-ch** [comment]
@magento run Static Tests

**Comment by kate-kyzyma** [comment]
@magento run all tests

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-8917](https://jira.corp.magento.com/browse/ENGCOM-8917) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
@kate-kyzyma, Please remove the conflicts on the PR

**Review by engcom-Delta — CHANGES_REQUESTED** [review]
Remove the existing conflicts

**Comment by engcom-Dash** [comment]
Closing this PR since it has not been updated for a long time. Please feel free to reopen it if needed.

**Pull request closed without merging**